src/tests/ftest/launch.py

```diff
@@ -708,10 +732,25 @@ def replace_yaml_file(yaml_file, args, tmp_dir):
             cmd = ["diff", "-y", orig_yaml_file, yaml_file]
             print(get_output(cmd, False))
 
+
     # Return the untouched or modified yaml file
     return yaml_file
 
 
+def generate_certs():
+    """function to generate the certificates for the test.
+
+    Args: none
+    Returns: none
+
+    """
+    # Generate certificates if not exist
+    print(" ")
+    if not os.path.exists("./daosCA"):
+        subprocess.call(["../../../../../utils/certs/gen_certificates.sh"])
```

Comment:
Will this path be correct if installed by RPMs?  I suspect not.  Your probably want to base this path on `PREFIX`.

You can test in CI with RPMs by adding `RPM-test: true` to your commit message.

Reply:
Secure by default is the final goal. 
We want to turn on the certificates for our functional tests gradually, or the failures will be unpredictable and overwhelm.
Currently, all the certificates tests were done manually, I choose 11 functional tests as pilot-test with certificates for CI.
At this phase, the location of the certificates is not critical.
We want to see the stability of the 11 functional tests with certificates.

Reply:
Even if we want to enable it gradually (I assume you mean over time, even once the initial patch lands), the patch that lands (i.e. this patch) that enables the feature should still be **secure by default** and the tests that you don't want to enable security on should have it specifically disabled, rather than the opposite stance that this patch is taking.

Reply:
When we turn on the secure-mode on functional tests, we want to make sure the tests are validated and stabilized, that's the pilot tests with certificates for.

Reply:
So to be clear, you are not seeking to land this PR as is, using **insecure by default** mode, is that correct?

Reply:
Insecure-mode is currently in default, and this PR is to enable the certificates with 11 pilot tests for CI.

Reply:
OK.  I will maintain my -1 at least until **secure by default** is enabled.